marklogic-data-hub/src/test/java/com/marklogic/hub/HubTestBase.java

```diff
@@ -340,67 +360,110 @@ protected void disableTracing() {
 
 		});
     }
-
-    protected HubConfig getHubConfig() {
-        return getHubConfig(PROJECT_PATH);
+    //'requireAdmin' indicates whether hubconfig uses 'hub-admin-user' or not ('data-hub-user' is used).
+    protected HubConfig getHubConfig(boolean requireAdmin) {
```

Comment:
Can you think of a different way to expose this than as a boolean flag?  I'm not sure if it's easy, but an overload like getAdminHubConfig() might be cleaner looking.  It should be very clear to the future reader whether any particular connection is using admin or not.